src/messages/messagesNB.ts

```diff
@@ -202,6 +203,10 @@ const messages = {
     title: "Tilgang nektet",
     errorDescription: "Du har ikke tilgang til denne siden.",
   },
+  collectionPage: {
+    title: `Ressurser på $t(languages.{{language}})`,
```

Comment:
Skal ikkje disse være slik?
```suggestion
    title: `Ressurser på {{language}}`,
```